### PR TITLE
Add Domainname.shop

### PR DESCRIPTION
**Pull request by mdavidsen**
domainname.shop operates under several domains, domene.shop (Norway), domän.shop (Sweeden), domæne.shop (Denmark). 
The following characters are allowed: `A-Z, a-z, 0-9 and + - * / ! ? . , : ; = # @ $ % & ( ) < >`, password lenght 10-79 chars

### CODE REVIEW

File: src/_data/sites/domainname.shop.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+description: |-
+    domainname.shop operates under several domains, domene.shop (Norway), domän.shop (Sweeden), domæne.shop (Denmark).
+    The following characters are allowed: A-Z, a-z, 0-9 and + - * / ! ? . , : ; = # @ $ % & ( ) < >, password lenght 10-79 chars
```
Comment (duffn) [inline src/_data/sites/domainname.shop.yaml:3]: Length is misspelled here.

File: src/_data/sites/domainname.shop.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+images:
+  - domainname.shop-password-rules.png
+  - domainname.shop-password-length.png
+name: domainname.shop - domain name registrar
+url: https://domainname.shop
```
Comment (duffn) [inline src/_data/sites/domainname.shop.yaml:8]: Nit: can you add a new line at the end of the file please.

**Review by duffn — CHANGES_REQUESTED** [review]
Thanks, just a few notes!

**Comment by mdavidsen** [comment]
Commets fixed and pushed :)

**Review by duffn — APPROVED** [review]

**Comment by duffn** [comment]
Thank you for the contribution!

**Pull request merged**